doc/ipython-notebooks/gaussian_process/variational_classifier.ipynb

```diff
@@ -1279,7 +1278,7 @@
     "    name=inf.get_name()\n",
     "    prbs=gpc.get_probabilities(features_boundary)\n",
     "    plot_helper(plots[0],x1_boundary,x2_boundary,prbs,n_boundary,name,x1s,x2s,\"without optimizing %d inducing points\"%n_inducing) \n",
-    "    x_inducing=inf.get_inducing_features().get_computed_dot_feature_matrix()\n",
+    "    x_inducing=inf.get_inducing_features().get(\"feature_matrix\")  \n",
```

Comment:
@vigsterkr how do you call `get_computed_dot_feature_matrix` with a Features object?

Reply:
pretty sure what I wrote here isn't right

Reply:
have a look at the plot, that makes it clear
I actually think this might work